tasks/serializers.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework import serializers
+from .models import Task
+
+
+class CreateTaskSerializer(serializers.ModelSerializer):
+
+    share_link = serializers.CharField(default=None)
+    share_text = serializers.CharField(default=None)
+    share_img = serializers.ImageField(default=None)
+
+    class Meta:
+        model = Task
+        fields=['social_media_platform', 'share_type', 'share_link', 'share_text', 'share_img']
```

Comment:
You forgot description field ...or is it not necessary? :laughing:

Reply:
Lets keep it like, you revealed an disregarded necessity 🤭🤝

Reply:
My pleasure 😂😂